packages/shared/src/components/auth/AuthHeader.tsx

```diff
@@ -1,15 +1,19 @@
-import React, { ReactElement } from 'react';
+import React, {
+  FormEvent,
+  MouseEvent,
+  KeyboardEvent,
+  ReactElement,
+} from 'react';
 import { Button } from '../buttons/Button';
 import ArrowIcon from '../icons/Arrow';
-import { CloseModalFunc } from '../modals/common';
 import { Modal } from '../modals/common/Modal';
 import { ModalHeaderKind } from '../modals/common/types';
 
 export interface AuthHeaderProps {
   simplified: boolean;
   title: string;
   className?: string;
-  onBack?: CloseModalFunc;
```

Comment:
Do we still use this somewhere?

Reply:
I just manually typed the prop to avoid the deps cycle being thrown, but I believe yes, we still use this property and type for simplicity of declaring the callback.

Reply:
I noticed in my vote refactor few deps cycles creeping up, we might wanna adopt `types.ts` as separate file for types to avoid it and also use `import type` vs just `import`. We can discuss on team meet?

Reply:
Yep. A good discussion to tackle in the Gathering/web team meeting. Have had this issue a few times already too.